.i18nrc.json

```diff
@@ -10,7 +10,8 @@
     "core": [
       "src/core",
       "packages/core/i18n/core-i18n-browser-internal",
-      "packages/core/fatal-errors/core-fatal-errors-browser-internal"
+      "packages/core/fatal-errors/core-fatal-errors-browser-internal",
+      "packages/core/deprecations/core-deprecations-browser-internal"
```

Comment:
Can't we just specify `"packages/core"` here? cc @Bamieh

Reply:
I'll see what CI says about the change in this PR since it's a relatively small one and doesn't involve other teams.